### ISSUE TITLE
Move view specific options to seperate section in settings

### ISSUE THREAD

**Issue by jordan1776** [thunder-app/thunder#242]
**Is your feature request related to a problem? Please describe.**
The "general" page in settings is beginning to get cluttered with settings that are marked to only work in specific view modes.

**Describe the solution you'd like**

All view specific options should be moved out of "general" to a "views" page. The only mention of views in "general" should be the "compact view" toggle . "General" should only have options that apply universally.

 One way of organizing the "views" page is  by having a top header for each view (compact, normal) and sub headers for organization  (feed, posts). The top headers could be made as accordians so the page would not get too tall when more views are added.

Basically
- Normal
  - Feed
  - Links
- Compact
  - Feed

**Describe alternatives you've considered**
None

**Additional context**
I think the "normal" view should be renamed to give new users a better context of what the default view is.

And also also the "compact view" toggle should probably be a drop-down to support more views.

![Screenshot_20230705-204524](https://github.com/hjiangsu/thunder/assets/6100101/97b0be3f-a718-41b1-a183-348a8dea0cbc)
